### DISCUSSION TITLE
Unsaved Settings get discarded

### DISCUSSION THREAD

**Discussion by moritztim** [dani-garcia/vaultwarden#4017] [Ideas]
* vaultwarden version: v1.29.2

### Steps to reproduce
1. Log into Admin page
2. Change a bunch of settings
3. Make a mistake such as setting an invalid SMTP auth mechanism
4. Click Save
### Expected behaviour
Usually in web forms, all the data stays and the relevant field is highlighted.
### Actual behaviour
You get a Popup telling you your one mistake and none of your changes are saved.
### Workaround
The only way to ensure this doesn't happen is to save after every change (so why not have autosave).
### Solution
Either make an autosave mechanism or keep the changes on the client or send them back.
If that isn't an option, at least having a save button per section would already minimize the data loss because this is particularily annoying when setting up vw for the first time and doing a whole bunch of config.

**Comment by BlackDex** [comment]
Going to move this to Ideas and place it on the Meta Feature Requests #246
